challenges/objects-arrays.js

```diff
@@ -83,43 +92,43 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 {"animal_name":"Hawk-eagle, crowned","population":10,"scientific_name":"Spizaetus coronatus","state":"Florida"},
 {"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
 
-/* Request 1: .forEach()
-
-The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
+/* Request 1: .forEach()
+The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and 
+scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 */
 const animalNames = [];
+zooAnimals.forEach((element) => animalNames.push(`Name: ${element.animal_name}, Scientific: ${element.scientific_name}`));
```

Comment:
I like the one liner, very clean. Only recommendation is maybe to use a more semantic parameter in your callback function in the forEach. zooAnimals is an array of objects, and the elements your looping over are objects that represent a single animal.
```suggestion
zooAnimals.forEach((animal) => animalNames.push(`Name: ${animal.animal_name}, Scientific: ${animal.scientific_name}`));
```